Src/Core/CaseStudy.Core/Models/Cars.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace CaseStudy.Core.Models
+{
+    public class Cars
+    {
+        [Key]
+        public int HomeNetVehicleId { get; set; }
+
+        [Required]
+        public int DealerId { get; set; }
+
+        [Required]
+        [StringLength(255)]
+        public string Type { get; set; }
+
+        [Required]
+        [StringLength(255)]
+        public string VIN { get; set; }
+
+        [Required]
+        public int Year { get; set; }
+
+        [Required]
+        [StringLength(255)]
+        public string? Make { get; set; }
```

Comment:
20 length, 
what happens if we remove ? here, is it mandatory? why?

Reply:
? is added for handling nulls, if removed , in the operations performed in further might get exceptions